server/rest_slicer_cli.py

```diff
@@ -867,24 +1019,25 @@ def genRESTEndPointsForSlicerCLIsInDocker(info, restResource, dockerImages):
     for dimg in dockerImages:
```

Comment:
In lines 999-1006 you do 

```python
    if isinstance(dockerImages, list):
        for img in dockerImages:
            if not isinstance(img, str):
                raise Exception('dockerImages must either be a single '
                                'docker image string or a list of docker '
                                'image strings')
    else:
        dockerImages = [dockerImages]
```

I feel this should not be done here. The output of docker images obtained from 
```python
dockerImages = dockerCache.getDockerImageList()
```
should already have been checked for errors inside. So, move that sanity check somewhere inside dockerCache.getDockerImageList() or elsewhere within DockerCache where list of docker image strings is provided as input to DockerCache.

Also, let us assume that `dockerCache.getDockerImageList` returns a list even if there is a single docker image.